dependencies.yaml

```
@@ -241,17 +242,23 @@ dependencies:
       - output_types: conda
         packages:
           - &cupy_unsuffixed cupy>=13.6.0
+          - libnvimgcodec == 0.6.0
```

Would make to drop spaces around the operators

```suggestion
          - libnvimgcodec==0.6.0
```

---

thank you for the review @jakirkham - will apply the suggestions one by one